src/python/tritonfrontend/CMakeLists.txt

```diff
@@ -0,0 +1,126 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+cmake_minimum_required (VERSION 3.18)
+
+project(tritonfrontend-bindings LANGUAGES C CXX)
+
+# ================= Ensures Package is Structured Properly ==================
+# Top level module entry point and typed marker
+file(COPY __init__.py DESTINATION .)
```

Comment:
I would like to understand better those instructions...  
1. Why Python files are copied over by CMake..?

Reply:
These CMake instructions were copied over from the existing [`CMakeLists.txt`](https://github.com/triton-inference-server/core/blob/main/python/tritonserver/CMakeLists.txt) for the core python-bindings.

From my limited understanding, I believe that these commands ensure that the python files are in the correct location when the python wheel is being built.

@GuanLuo might be able to provide more insight.

Reply:
No specific intention here - I don't believe the exact cmake instructions are critical - though did have to play around with the way the files were globbed ... 

As long as the end result is correct - am ok with whatever best practice is suggested - we should update the other to match - 

I think can be done post first release - / seperate MR - if the existing is working